### PR TITLE
Transfered psycopg2 codes to query and result

### PR DESCRIPTION
**Pull request by JosiahRegencia**
For issue #117

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ services:
       retries: 3
     # UN-COMMENT for local testing:
     #ports:
-    #  - "5432:5432"
+      #- "5432:5432"
```
Comment (greenmoss) [inline docker-compose.yml]: Please revert this to the previous value. This sounds nit-picky,  but here's the reason: when looking through code diffs between commits, we want the "significant" changes to be called out. This change is not functionally different, so it is distracting when viewing the diffs.

File: savant/app/query.py [hunk 2]
```diff
@@ -153,3 +153,17 @@ def new_users(self):
...
         return self.new_users
+
+    def null_cases(self):
+        self.cur.execute("select store_debs(%s, %s ,%s, %s)", ('stat', None, 'test', 'test'))
+        self.conn.commit()
+        self.cur.execute("select store_debs(%s, %s ,%s, %s)", ('stat', None, 'test', 'test'))
+        self.conn.commit()
+        self.cur.execute("select count(*) from debs where stat=%s and name is null and version=%s and architecture=%s", ('stat', 'test', 'test'))
+        count = self.cur.fetchone()
+        # self.assertEqual(count[0], 1)
```
Comment (greenmoss) [inline savant/app/query.py]: For testing, `assert` statements are usually more readable and understandable than `if count...`. 

If possible, please un-comment this and remove lines 166-169.

If that is not possible, please remove this commented line of code.

**Review by greenmoss — CHANGES_REQUESTED** [review]

File: savant/app/query.py (hunk 2 above)
Comment (JosiahRegencia) [reply savant/app/query.py]: assert only works in unit testing I guess

Comment (greenmoss) [reply savant/app/query.py]: I noticed this is `self.assert`. Does plain old `assert` work?

File: docker-compose.yml (hunk 1 above)
Comment (JosiahRegencia) [reply docker-compose.yml]: I'm not really sure what you mean by this one. I just uncommented it and then commented it again.

Comment (JosiahRegencia) [reply docker-compose.yml]: Latest commit in master:

https://github.com/Doveps/mono/blob/fe2fb79bf1d01cd81f1abd6f4057dee03565942b/docker-compose.yml#L17-L18

Comment (JosiahRegencia) [reply docker-compose.yml]: https://github.com/Doveps/mono/blob/fe2fb79bf1d01cd81f1abd6f4057dee03565942b/docker-compose.yml#L17-L18


That's the last commit from the master. I'm not really sure what values, though.

File: savant/app/query.py (hunk 2 above)
Comment (JosiahRegencia) [reply savant/app/query.py]: I'm goign to try it

**Review by JosiahRegencia — COMMENTED** [review]
Changed it to assert:

https://github.com/Doveps/mono/blob/9ef89fe8813962d336a6b1f0829cde0caf46378e/savant/app/query.py#L157-L165

**Comment by JosiahRegencia** [comment]
https://travis-ci.org/Doveps/mono/builds/349672518

**Review by greenmoss — APPROVED** [review]

**Comment by greenmoss** [comment]
Awesome, thanks!

**Pull request merged**